src/MSBuildLocator/MSBuildLocator.cs

```diff
@@ -239,7 +230,7 @@ public static void Unregister()
             if (!IsRegistered)
             {
                 var error = $"{typeof(MSBuildLocator)}.{nameof(Unregister)} was called, but no MSBuild instance is registered." + Environment.NewLine;
-                if (numResolvedAssemblies == 0)
+                if (!registerCalled)
```

Comment:
This error message is now wrong. Unregister isn't called automatically, so the only way IsRegistered would be false but registerCalled would be true is if they manually Unregistered twice, which probably isn't a big thing. I'd prefer to unset registerCalled here and keep just the one message.

Reply:
I'm not 100% sure I understand what you mean but go ahead and do that.